### ISSUE TITLE
Completion doesn't work on same file

### ISSUE THREAD

**Issue by faustinoaq** [crystal-lang-tools/scry#97]
Hi @laginha87 I just found another little issue with completion:

When I was testing #92 I realize the completion follows requires but doesn't includes content for the file itself.

By example:

This works:

![image](https://user-images.githubusercontent.com/3067335/39413343-59dcde08-4bee-11e8-968c-25ae3addbdd4.png)

This doesn't work:

![image](https://user-images.githubusercontent.com/3067335/39413350-6b8d0830-4bee-11e8-8c76-d187cacb6b1f.png)

I guess there is some issue with requires parsing somewhere here:

https://github.com/crystal-lang-tools/scry/blob/9b1745a8ad536598d22676ae665a136c0b392c4a/src/scry/completion/dependency_graph.cr#L83-L85

Are we missing the current file content on completion?

**Comment by laginha87** [comment]
You're right it doesn't add the file itself when generating it's own method db. I was thinking of a one class per file style when I created that and didn't think it was a good idea, but yeah we should add it.

**Comment by faustinoaq** [comment]
@laginha87 Now #61 is already fixed by #98 , so this is the last issue to make current completion features almost perfect  :tada:  :sparkles:

**Comment by faustinoaq** [comment]
> I was thinking of a one class per file style when I created that and didn't think it was a good idea, but yeah we should add it.

So, Do you have any idea how to fix it? :sweat_smile:
